backend/src/main/java/de/jannisaziz/backend/security/LoginController.java

```diff
@@ -0,0 +1,57 @@
+package de.jannisaziz.backend.security;
+
+import de.jannisaziz.backend.user.UserDTO;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.servlet.http.HttpServletResponse;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/api/auth")
+public class LoginController {
+
+    private final LoginService loginService;
+
+    @PostMapping("signIn")
+    public UserDTO signIn(@RequestBody LoginRequest request) throws ResponseStatusException {
+        try {
+            return loginService.signIn(request).asDTO();
+        } catch (IllegalStateException e) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, e.getMessage());
+        }
+    }
+
+    @PostMapping("signUp")
+    public String signUp(@RequestBody LoginRequest request) throws ResponseStatusException {
```

Comment:
Maybe return just a bool or a singupRresponseDto with emailConfirmation: true and the email: <user.email>?